benchmarks/benchmark/tools/dlio/README.md

```diff
@@ -16,12 +16,19 @@ If you need to reinstall any resources, make sure to delete this file as well.
 
 ## Run DLIO Job
 1. Update the `variables.tf` file with your desired settings to run your machine learning benchmark workload
-2. Change the dlio image in `dlio/podspec.tpl`
+2. Change the dlio image in `modules/dlio/podspec.tpl`
 3. Run `terraform init`
 4. Run `terraform apply`
 5. After you finish your test, run `terraform destroy` to delete the
    resources
 
+## Run DLIO Job with Parallelstore
```

Comment:
please mention what are the pre-reqs here before deploy of the dlio job e.g network peering

Reply:
done.